components/setting_picture.jsx

```diff
@@ -227,7 +225,7 @@ export default class SettingPicture extends Component {
             fileInputDisabled = true;
         } else {
             let confirmButtonClass = 'btn btn-sm';
-            if (this.props.submitActive) {
+            if (this.props.submitActive || this.state.removeSrc) {
```

Comment:
Is there an opportunity for another snapshot unit test to test the new behaviour?